Source/Orts.Simulation/Simulation/Activity.cs

```diff
@@ -1334,7 +1334,7 @@ override public Boolean Triggered(Activity activity)
                     {
                         consistTrain = null;
```

Comment:
Side note, can't we delete this line? It's a redundant assignment.

Reply:
OK, line removed together with a similar nearby one.